code/game/machinery/computer/arcade.dm

```diff
@@ -1273,9 +1297,9 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 		var/obj/item/bodypart/chopchop = c_user.get_bodypart(which_hand)
 		chopchop.dismember()
 		qdel(chopchop)
+		user?.mind.adjust_experience(/datum/skill/gaming, 100)
```

Comment:
Better yet would there be any harm in there being a chance to avoid having your arm get taken by the machine if you're a skilled gamer? Basically the Handless millionaire game except you're risking a limb for some mere toys (Unless someone was offering credits if you won).

Reply:
It's fine